frontend/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
Is this a comment?

Reply:
It's a typescript compiler directive. I'm not sure it's needed unless you are customising react scripts I think. It's early in the project so I'll leave it around.